rsocket-core/src/commonMain/kotlin/io/rsocket/kotlin/frame/CancelFrame.kt

```diff
@@ -22,5 +22,5 @@ class CancelFrame(
     override val streamId: Int
 ) : Frame(FrameType.Cancel) {
     override val flags: Int get() = 0
-    override fun Output.writeSelf(): Unit = Unit
+    override fun BytePacketBuilder.writeSelf(): Unit = Unit
```

Comment:
TIL: This is one of the least intuitive bits of code to read, but makes sense now.